src/main/java/io/debezium/connector/vitess/VitessErrorHandler.java

```diff
@@ -20,7 +18,15 @@ public VitessErrorHandler(String logicalName, ChangeEventQueue<?> queue) {
     protected boolean isRetriable(Throwable throwable) {
         if (throwable instanceof StatusRuntimeException) {
             final StatusRuntimeException exception = (StatusRuntimeException) throwable;
-            return exception.getStatus().getCode().equals(UNAVAILABLE);
+            String description = exception.getStatus().getDescription();
+            switch (exception.getStatus().getCode()) {
+                case UNAVAILABLE:
+                    return true;
+                case UNKNOWN:
+                    if (description != null && description.equals("stream timeout")) {
+                        return true;
+                    }
```

Comment:
nit: Might worth adding a comment saying why this should be retryable (basically what you have in the PR description :))